File: 07_covid home/200317_Collection/example#3/java/before.java
public class Control
{
    
	public void forward() {

		System.out.println("move forward");
	}
	
	public void backward() {

		System.out.println("move backward");
	}
	
	public void left() {

		System.out.println("move left");
	}
	
	public void right() {

		System.out.println("move right");
	}


	public void move(char key) {
	    
	    
		if(key == 'w'){
			forward();
	    }
		else if(key == 's'){
			backward();
		}
		else if(key == 'a'){
			left();
		}
		else if(key == 'd'){
			right();
		}
	}


	public static void main(String[] args) {
		
        Control Control = new Control();
	        
        Control.move('w');
        Control.move('a');
	}
}

File: 07_covid home/200317_Collection/example#3/python/before.py
def forward():
    print('move forward')

def left():
    print('move left')
    
def right():
    print('move right')

def backward():
    print('move backward')


def move(key):
    
    dict = {'w':forward, 'a':left, 'd':right, 's':backward}
    
    func = dict.get(key)
    
    if func is not None:
        func()
    

key = 'w'

move(key)
